tools/wpt/expectation.json

```diff
@@ -6219,231 +6149,21 @@
-        "ReadableStream with byte source: read(view) with Uint16Array on close()-d stream with 1 byte enqueue()-d must fail",
-        "ReadableStream with byte source: A stream must be errored if close()-d before fulfilling read(view) with Uint16Array",
-        "ReadableStream with byte source: read(view), then respond() and close() in pull()",
-        "ReadableStream with byte source: read(view) with Uint32Array, then fill it by multiple respond() calls",
-        "ReadableStream with byte source: read(view) with Uint32Array, then fill it by multiple enqueue() calls",
-        "ReadableStream with byte source: read() twice, then enqueue() twice",
-        "ReadableStream with byte source: Multiple read(view), close() and respond()",
-        "ReadableStream with byte source: Multiple read(view), big enqueue()",
-        "ReadableStream with byte source: Multiple read(view) and multiple enqueue()",
-        "ReadableStream with byte source: read(view) with passing undefined as view must fail",
-        "ReadableStream with byte source: read(view) with passing an empty object as view must fail",
-        "ReadableStream with byte source: Even read(view) with passing ArrayBufferView like object as view must fail",
-        "ReadableStream with byte source: read(view) on an errored stream",
-        "ReadableStream with byte source: read(view), then error()",
-        "ReadableStream with byte source: Throwing in pull function must error the stream",
-        "ReadableStream with byte source: Throwing in pull in response to read() must be ignored if the stream is errored in it",
-        "ReadableStream with byte source: Throwing in pull in response to read(view) function must error the stream",
-        "ReadableStream with byte source: Throwing in pull in response to read(view) must be ignored if the stream is errored in it",
-        "calling respond() twice on the same byobRequest should throw",
-        "calling respondWithNewView() twice on the same byobRequest should throw",
-        "calling respond(0) twice on the same byobRequest should throw even when closed",
-        "calling respond() should throw when canceled",
-        "pull() resolving should not make releaseLock() possible",
-        "ReadableStream with byte source: default reader + autoAllocateChunkSize + byobRequest interaction",
-        "ReadableStream with byte source: respondWithNewView() with a smaller view",
-        "ReadableStream with byte source: respondWithNewView() with a zero-length view (in the closed state)",
-        "ReadableStream with byte source: respondWithNewView() with a transferred non-zero-length view (in the readable state)",
-        "ReadableStream with byte source: respondWithNewView() with a transferred zero-length view (in the closed state)"
+        "ReadableStream with byte source: Respond to multiple pull() by separate enqueue()"
```

Comment:
After investigation I am about 80% confident this test being broken is a result of our broken microtask queuing: https://github.com/denoland/deno/issues/11731